sources/PTYTextView.m

```diff
@@ -4831,8 +4833,10 @@ - (void)doCommandBySelector:(SEL)aSelector
         // See comment in -keyDown:
         DLog(@"Rerouting doCommandBySelector from %@ to %@", self, gCurrentKeyEventTextView);
         [gCurrentKeyEventTextView doCommandBySelector:aSelector];
+        [self.delegate keyDown:_keydownEvent];
         return;
     }
+    [self.delegate keyDown:_keydownEvent];
```

Comment:
Really? I reached here when press enter key.

Would you talk about line 4836(https://github.com/gnachman/iTerm2/pull/279/files#diff-9b0660c8ca6c9370c3a473d7c2b172b8R4836)?